iina/KeyMapping.swift

```diff
@@ -34,15 +34,37 @@ class KeyMapping {
     "NEXT": "▶︎▶︎"
   ]
 
+  var isIINACommand: Bool
+
   var key: String
 
   var action: [String]
 
+  private var privateRawAction: String
+
+  var rawAction: String {
+    set {
+      if newValue.hasPrefix("@iina") {
+        privateRawAction = newValue.substring(from: newValue.index(newValue.startIndex, offsetBy: 5)).trimmingCharacters(in: .whitespaces)
```

Comment:
`5` looks like a magic number here; I'd use `"@iina".count` here to make the relationship explicit